manifests/e/EricLawrence/FiddlerNetLogImporter/1.3.4.4/EricLawrence.FiddlerNetLogImporter.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: EricLawrence.FiddlerNetLogImporter
+PackageVersion: 1.3.4.4
+PackageLocale: en-US
+Publisher: Eric Lawrence
+PackageName: Fiddler NetLog Importer
+License: Open source
+Copyright: ©2023 Eric Lawrence
+ShortDescription: Installer for Fiddler Fiddler NetLog Importer

```

Comment:
Please provide a description of what Fiddler NetLog Importer does instead of saying it is just the installer